src/ui/ResultsPerPage/ResultsPerPage.ts

```diff
@@ -15,7 +15,11 @@ export interface IResultsPerPageOptions {
 }
 
 /**
- * This component attaches itself to a div and allows users to choose the number of results displayed per page.<br/>
+ * The ResultsPerPage component attaches itself to a `div` and allows the end user to choose how many results to
+ * display per page.
+ *
+ * **Note:** Adding a ResultPerPage component to your page overrides the value of
```

Comment:
I found this out while testing; please don't hesitate to tell me if I'm wrong! :)